code/modules/mob/living/abilities/gallop.dm

```diff
@@ -50,9 +50,8 @@
 		GLOB.moved_event.register(user, src, /datum/extension/gallop/proc/user_moved)
 
 /datum/extension/gallop/proc/destroy()
```

Comment:
i think i've messed up a bit here, this should probably be Destroy, and it should call parent